### ISSUE TITLE
[Feature] Admin Pannel

### ISSUE THREAD

**Issue by 6543** [woodpecker-ci/woodpecker#345]
add an Admin panel to:
 * see webhooks
 * see all activated repos
 * see all registerd agents
 * ...

**Comment by jwildeboer** [comment]
Some basic graphs that show jobs per hour/day/week (maybe per queue), average and min/max runtime/memory/storage use of jobs, Number of fails/cancels/OK per hour/day/week, most often running jobs, most jobs per user/org

**Comment by JulienPalard** [comment]
I'm loving the admin panel, but it looks like the "last_contact" field is never updated. I'm running v1.0.5 (both server and agent side). My agent correctly picks the jobs though, so they're in contact.

Now for example I see "2 hours ago" for the last contact. I checked the sqlite db and it looks like it has been updated at least once a few minutes after its creation:

```
sqlite> select last_contact - created from agents;
261
```

I tried to restart the agent but ... oh it created a new line in the agents table, I was not expecting this.

**Comment by 6543** [comment]
@JulienPalard try using `next` (upcomming v2.0.0)

**Comment by JulienPalard** [comment]
Oh, looks like it gets updated, just slowly, I now see "last contact: 4 hours ago", which means the last_contact gets updated, but not that often (like every 4 hours ?).

@6543 I'll gladly beta-test the v2.0.0, should I also upgrade my agents?

**Comment by 6543** [comment]
yes you have to update server and agents at the same time ... else agents wont connect anymore
